projects/plugins/jetpack/wp-cli-templates/block-edit-js.mustache

```diff
@@ -8,7 +8,7 @@ import './editor.scss';
 
 const icon = getBlockIconComponent( metadata );
 
-function {{ className }}Edit( { attributes, className, noticeOperations, noticeUI, setAttributes } ) {
```

Comment:
In this API version, `className` is not passed to the component props anymore. Instead, it's present in `blockProps`.